### DISCUSSION TITLE
Laravel 9 Plans for Tenancy

### DISCUSSION THREAD

**Discussion by decassidy** [tenancy/tenancy#247] [Q&A]
Where are you planning to update Tenancy to work with the Laravel version 9?

**Comment by thumbtech** [comment]
FWIW mostly the devs are in [discord](https://tenancy.dev/chat) and there they have stated they do plan on supporting v9.

**Reply by sniper7kills** [reply]
That is certainly not the case; Currently the main dev's are extremally busy in their personal lives; If I could merge my PR; I certainly would.

**Reply by decassidy** [reply]
> That is certainly not the case; Currently the main dev's are extremally busy in their personal lives; If I could merge my PR; I certainly would.

That's not a great move not to prioritize changes to keep tenancy up to date especially if you are seeking sponsorship and funding from the community. Laravel makes major changes every February to give their users a specific timeline to expect upgrades.
